ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -151,7 +151,7 @@ const DropdownBulkSelect = ({
           key="select-multiple-split-checkbox"
           id="select-multiple-split-checkbox"
           aria-label={_("Select multiple disks")}
-          isChecked={selectedCnt > 0 ? (selectedCnt === totalCnt ? true : null) : false}
+          isChecked={selectedCnt === totalCnt}
```

Comment:
So we want this UI change - to get rid of this bulk selection checkbox state (some disks (neither all nor none) selected in this PR)?
![Screenshot from 2023-07-13 09-47-37](https://github.com/rhinstaller/anaconda/assets/1220237/eb60047f-807a-4ab9-aed8-94fc6d574687)

Reply:
oops good catch, how the tests even passed here? I will remove this commit.

Reply:
I think the checkbox should be indeed only checked if all are checked.

Reply:
> oops good catch, how the tests even passed here? I will remove this commit.

I am afraid I just didn't write a test for this case. I'll add it.